packages/database/src/migrations/20220218050602-AddCovidKiribatiProject-modifies-data.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+import { insertObject, codeToId, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const PROJECT_NAME = 'COVID-19 Kiribati';
+const PROJECT_CODE = 'covid_kiribati';
+const DASHBOARD_CODE = 'covid_ki';
+const DASHBOARD_NAME = 'COVID-19 Kiribati';
+const MAP_OVERLAY_GROUP_CODE = 'covid_ki';
+const MAP_OVERLAY_GROUP_NAME = 'COVID-19 Kiribati';
+const PROJECT = {
+  code: PROJECT_CODE,
+  description: 'COVID-19 case surveillance and data visualisations in Kiribati',
+  sort_order: 16,
+  image_url:
+    'https://tupaia.s3.ap-southeast-2.amazonaws.com/thumbnails/uploads/covid_kiribati_background.jpeg',
```

Comment:
The `thumbnails` folder on S3 is actually supposed to be populated with smaller versions of images uploaded to the main `uploads` folder (there's a lambda script that detects the upload and puts a compressed version in `thumbnails/uploads`)

Reply:
aha! Thanks.